datumaro/util/definitions.py

```diff
@@ -2,8 +2,15 @@
 #
 # SPDX-License-Identifier: MIT
 
+import os
+import os.path as osp
 from typing import Tuple
 
 DEFAULT_SUBSET_NAME = "default"
 BboxIntCoords = Tuple[int, int, int, int]  # (x, y, w, h)
 SUBSET_NAME_BLACKLIST = {"labels", "images", "annotations", "instances"}
+
+DATUMARO_CACHE_DIR = os.path.join(os.path.expanduser("~"), ".cache", "datumaro")
```

Comment:
> 
```suggestion
DATUMARO_CACHE_DIR = os.path.expanduser(os.path.join(os.getenv("XDG_CACHE_HOME", "~/.cache"), "datumaro"))
```

Complying Linux `XDG directory` [specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html)

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/1032/commits/9238190d1fab8381e3ba9c2e5373d19c282c3e93